.github/workflows/test.yml

```diff
@@ -40,14 +40,29 @@ jobs:
           sudo apt update
           sudo apt install libgdcm-dev
 
+      # install from source, because apt has old version
+      - name: Install GoogleTest and GoogleMock
+        run: |
+          git clone https://github.com/google/googletest.git -b v1.13.0
```

Comment:
Why don't we put that in the CMake itself ?

https://google.github.io/googletest/quickstart-cmake.html

Personally I don't like hashes, but maybe we could use tag name of latest release ?



![image](https://github.com/DataMedSci/yagit/assets/7374733/c5aa9cb9-71fb-4f46-97e1-0afefc681c9d)

Reply:
I didn't use that method, because then yagit would require all users to build GoogleTest from source. Now users can download built version of GoogleTest (e.g. on Windows using Conan, on Linux using apt).
I build it from source only in GitHub Actions workflow. That's because I set the Ubuntu version to "ubuntu-20.04", which is a version from 2020. I set this version to check if yagit can be run on older version of Ubuntu. Unfortunately it contains old version of GoogleTest (1.10). But if someone has newer version of ubuntu then he can download this package using apt and he will get newer version.

Reply:
It would be good to improve that in the future and have "matrix tests" which cover range of Ubuntu distribution versions. Then you could conditionally build from sources or use version available via apt. Again - a new issue.

Reply:
https://github.com/DataMedSci/yagit/issues/59